canonical/canonical.com
# Update contact form success banner

## Done
- Format files with djlint (mainly what's causing the large diff)
- Change form submission redirect from `#success` to `#contact-form-success`
- Shows the correct message: **Thank you for contacting our team. We will be in touch shortly.** as seen in the forms copy doc


## QA
- Go to demos: [/solutions/ai](https://canonical-com-1267.demos.haus/solutions/ai), [/solutions/secure-open-source](https://canonical-com-1267.demos.haus/solutions/secure-open-source), [/solutions/big-data-ai](https://canonical-com-1267.demos.haus/solutions/big-data-ai)
- Click on `Contact Canonical` and submit form
- On form submission success, check that url has `#contact-form-success` and is showing the correct notification banner (in screenshot below)
 - Forms copy docs: [secure-open-source](https://docs.google.com/document/d/10SsItwbzMqZB4NzKANiOW9kx05j5QXH7wUoD269jebY/edit), [ai](https://docs.google.com/document/d/1rlAO5gLtxsmi3la_yk2rCBOTXBAph2pkRhwVyu6Ua-o/edit)
 - Form copy docs for [big-data-ai](https://docs.google.com/document/d/1ZDYZHdATF7Pqlmkq9s-ucN9YdCu7ZA648vlGkyQFApc/edit) does not specify a confirmation banner message, but changed for consistency


## Issue / Card

Fixes [WD-11848](https://warthogs.atlassian.net/browse/WD-11848)

## Screenshots
![Screenshot 2024-06-05 at 2 25 23 PM](https://github.com/canonical/canonical.com/assets/62298176/73c8c994-55c3-4a44-969d-667626ee84db)




[WD-11848]: https://warthogs.atlassian.net/browse/WD-11848?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
